File: tests/workflow/consumer/rust/src/consumer/broadcasts/structc.rs
use super::{protocol, Consumer, Context};
use crate::stat::Alias;
use clibri::client;

pub async fn handler<E: client::Error>(
    event: protocol::StructC,
    context: &mut Context,
    consumer: Consumer<E>,
) {
    context.inc_stat(Alias::StructC);
}

File: experimental/workflow_mt/producer/typescript/src/producer/responses/structc.ts
import { Response } from "../implementation/responses/structc";
import {
    Context,
    Producer,
    Identification,
    Filter,
    Protocol,
} from "../implementation/responses";

export function response(
    request: Protocol.StructC,
    consumer: Identification,
    filter: Filter,
    context: Context,
    producer: Producer
): Promise<Response> {
    return Promise.reject(
    	new Error(`Handler for Protocol.StructC isn't implemented.`)
    );
}